src/server/system_services/system_store.js

```diff
@@ -742,7 +742,7 @@ class SystemStore extends EventEmitter {
         for (const res of bulk_results) {
             if (res && !res.ok) {
                 dbg.error('got error on bulk execute', res.err);
-                // should we throw here? retry?
+                throw new Error(res.err);
```

Comment:
@dannyzaken @guymguym What do you think? (please also comment in the PR)

Reply:
for now, I think we should throw here. 
later on, we should change the behavior to perform all bulk changes in the system_store in a single transaction (and not per collection)